routers/guests.py

```diff
@@ -0,0 +1,97 @@
+from typing import List
+from fastapi import APIRouter
+from schemas import guests as guests_schema, requests as requests_schema
+
+router = APIRouter()
+
+
+@router.get("/guests", response_model=List[guests_schema.GuestInfo])
```

Comment:
this could be one endpoint, if any filters provided -> apply them, else return all with limit and offset

Reply:
Okm will remove